core/jvm/src/main/scala/zio/sql/expr.scala

```diff
@@ -108,6 +108,7 @@ trait ExprModule extends NewtypesModule with FeaturesModule with OpsModule {
 
     def typeTagOf[A](expr: Expr[_, _, A]): TypeTag[A] = expr.asInstanceOf[InvariantExpr[_, _, A]].typeTag
 
+    implicit def literal(str: String): Expr[Features.Literal, Any, String] = Expr.Literal(s"'${str}'")
```

Comment:
Please remove this. We need to deal with at the `renderRead` level.

Reply:
You can add single quotes to the parameters in tests. We'll fix it once we fix `renderRead`.

Reply:
Done